src/include/daos/pool.h

```diff
@@ -78,8 +78,7 @@ struct dc_pool {
 	/* highest known pool map version */
 	uint32_t		dp_map_version_known;
 	uint32_t		dp_disconnecting:1,
-				dp_slave:1, /* generated via g2l */
-				dp_rf_valid:1;
```

Comment:
Intentional removal? Or bad merge?

Reply:
oh...dp_slave was a mistake.

Reply:
Ah, dp_rf_valid was removed on purpose, dp_slave was not removed..